SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -139,11 +140,18 @@ public Boolean Connect()
             return _rh.Init();
         }
 
-        public StatusResponse GetStatus()
+        /// <summary>
+        /// Gets the Spotify status
+        /// </summary>
+        /// <returns></returns>
+        public async Task<StatusResponse> GetStatus()
```

Comment:
Its probably because the `SynchronizingObject` of the timer is set to `this` it is running on the same thread.